### DISCUSSION TITLE
Mojo 🔥 Playground is being deprecated: March 20th, 2024

### DISCUSSION THREAD

**Discussion by ematejska** [modular/modular#1784] [Announcements]
Mojocians 👋 ,

To streamline our offerings given we've released the downloadable [Mojo SDK which supports Jupyter notebooks](https://github.com/modularml/mojo/tree/main/examples/notebooks?utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz--6GcUxZre86pkNY9kC157tf7CWUKjOdAQDZ4GVAACRxq7_PWhxZj-qZ517bugDzHzN3u0c#readme), **we will be turning down the [Mojo Playground](https://playground.modular.com/?utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz--6GcUxZre86pkNY9kC157tf7CWUKjOdAQDZ4GVAACRxq7_PWhxZj-qZ517bugDzHzN3u0c) (based on Jupyter) on March 20th, 2024.** If you have any notebooks you would like to save, please download them with the `File→Download` option we provided in the playground. We are also no longer prioritizing bugs that are specific to the Mojo Playground.

In the near future (yes, exciting! 🚀) we will also revamp our online documentation to provide interactive learning of the Mojo 🔥 language. This will support the most common use cases of the Mojo 🔥 Playground for quickly learning the basics of Mojo 🔥 without having to download anything locally.

To use example Jupyter notebooks, and experience the full power of Mojo  on your local machine - download the Mojo SDK [here](https://docs.modular.com/mojo/manual/get-started/).

Please feel free to post any concerns you have to the Github Discussion below!

Thanks for being part of our community! 🍻

**Comment by PlanetDestroyyer** [comment]
When we can use it in Windows without wsl??

**Reply by jayzhan211** [reply]
why not wsl?
